config/generateRunFile.coffee

```diff
@@ -429,6 +430,19 @@ generateDev = (KONFIG, options) ->
       fi
     }
 
+    function switch_client_version () {
+      if [ "$1" == "default" ]; then
+        rm $KONFIG_PROJECTROOT/CLIENTVERSION
+        pkill -SIGPIPE koding-webserver
+      elif [ -d "./website/a/p/p/$1" ]; then
```

Comment:
yes, I've removed the directory check since we can now use the reverse proxy over S3 for any compiled version.